rclcpp/test/rclcpp/test_time.cpp

```diff
@@ -352,3 +354,93 @@ TEST_F(TestTime, seconds) {
   EXPECT_DOUBLE_EQ(4.5, rclcpp::Time(4, 500000000).seconds());
   EXPECT_DOUBLE_EQ(2.5, rclcpp::Time(0, 2500000000).seconds());
 }
+
+TEST_F(TestTime, test_max) {
+  rclcpp::Time time_max = rclcpp::Time::max();
```

Comment:
Most of these rclcpp::Time variables you are setting up in your tests can be const. Fix here and below.

Reply:
You are right, fixed here: 095bd0f